receiver/googlecloudspannerreceiver/config_test.go

```diff
@@ -27,6 +27,10 @@ import (
 	"go.opentelemetry.io/collector/receiver/scraperhelper"
 )
 
+const (
+	cardinalityLimit = 200000
```

Comment:
Tip: you can use underscores to make the number more readable.

```suggestion
	cardinalityLimit = 200_000
```

Reply:
Done.